src/Engine/Graphics/ImageLoader.cpp

```diff
@@ -65,7 +62,7 @@ bool Paletted_Img_Loader::Load(RgbaImage *rgbaImage, GrayscaleImage *indexedImag
         return false;
 
     *indexedImage = GrayscaleImage::copy(tex->header.uTextureWidth, tex->header.uTextureHeight, tex->paletted_pixels);
-    *palette = MakePaletteAlpha(tex->pPalette24);
+    *palette = MakePaletteSolid(tex->pPalette24);
```

Comment:
It's only really ever used for buff icons, and those are processed in a way that all colors outside of a specific range are treated as transparent. I'll rewrite this part anyway, just not in this PR.

Reply:
To clarify - the change was needed because of the way palettes are now processed.

What we had before:
1. Palette was read as is.
2. When rendering indexed image into rgba image, pallete color 0 was replaced with transparent black. Even if it wasn't actually black in the original palette.

And thus we had an inconsistency there.

This inconsistency was fixed, and we got problems with palette-based animations, because in the original palette color at index zero wasn't transparent. So I did just that – reading palette as is.